### PR TITLE
null is not false

### PR DESCRIPTION
**Pull request by annando**
This is a quickfix for the latest PR.

### CODE REVIEW

File: include/features.php [hunk 1]
```diff
@@ -16,11 +16,11 @@
...
 
-	if ($x === false) {
+	if (is_null($x)) {
 		$x = PConfig::get($uid, 'feature', $feature);
-		if ($x === false) {
+		if (is_null($x)) {
 			$x = Config::get('feature', $feature);
-			if ($x === false) {
+			if (is_null($x)) {
 				$x = get_feature_default($feature);
```
Comment (MrPetovan) [inline include/features.php]: While we're at it, why not rewrite this as
````php
$x = Config::get('feature_lock', $feature, PConfig::get($uid, 'feature', $feature, Config::get('feature', $feature,  get_feature_default($feature))));
````
😇

**Comment by MrPetovan** [comment]
Related to #3873

File: include/features.php (hunk 1 above)
Comment (annando) [reply include/features.php]: This could be set on the list for later. This was the quickest and no so dirty clean up.

Comment (MrPetovan) [reply include/features.php]: This wasn’t serious! It would work, but it would be very inefficient and unreadable! 😅

Comment (annando) [reply include/features.php]: I really thought about it ;-) But I did a semi solution for the current PR.

**Pull request merged**